README.md

```diff
@@ -141,7 +141,7 @@ The following subjects are instructions for contributors who consider to submit
 ### Submit the changes
 
 You're all welcome to submit a pull request.
-Please consider the [pull request template](https://github.com/omise/omise-php/blob/master/.github/PULL_REQUEST_TEMPLATE.md) and fill the form when you submit a new pull request.
+Please consider the [pull request template](https://github.com/omise/.github/blob/main/pull_request_template.md) and fill the form when you submit a new pull request.
```

Comment:
I think we can just remove this line. What value does it add?

Reply:
True. Anyway, when a PR is created, the template is going to display to guide the submitter.